### PR TITLE
Create fr-FR.plg_system_customcss.sys.ini

### PR DESCRIPTION
**Pull request by AlainRnet**
french translation for your plugin. Thank you very much.

### CODE REVIEW

File: language/fr-FR/fr-FR.plg_system_customcss.sys.ini [hunk 1]
```diff
@@ -0,0 +1,6 @@
+; Custom CSS Plugin
+; Copyright (C) 2013 - 2020 Tobias Zulauf (jah-tz.de). All rights reserved.
+; License GNU General Public License, https://www.gnu.org/copyleft/gpl.html
+
+PLG_SYSTEM_CUSTOMCSS="Système - Ajouter un fichier CSS personnalisé"
+PLG_SYSTEM_CUSTOMCSS_XML_DESCRIPTION="Ce plugin vérifie si un fichier custom.css (ou custom.min.css) existe dans /templates/"templatename"/css ou dans /administrator/templates/"templatename"/css. Si oui, il sera chargé sur le site.<br>- Avec custom.css, vous pouvez écraser les règles des fichiers css du template.<br>- Ce fichier ne sera pas impacté lors d'une mise à jour.<br>- Pour utiliser le plugin, veuillez l'activer. Panneau d'administration --> Système --> Plugins --> Système - Ajouter un fichier CSS personnalisé"
```
Comment (zero-24) [inline language/fr-FR/fr-FR.plg_system_customcss.sys.ini]:
```suggestion
PLG_SYSTEM_CUSTOMCSS_XML_DESCRIPTION="Ce plugin vérifie si un fichier custom.css (ou custom.min.css) existe dans /templates/\"templatename\"/css ou dans /administrator/templates/\"templatename\"/css. Si oui, il sera chargé sur le site.<br>- Avec custom.css, vous pouvez écraser les règles des fichiers css du template.<br>- Ce fichier ne sera pas impacté lors d'une mise à jour.<br>- Pour utiliser le plugin, veuillez l'activer. Panneau d'administration --> Système --> Plugins --> Système - Ajouter un fichier CSS personnalisé"
```

Comment (zero-24) [reply language/fr-FR/fr-FR.plg_system_customcss.sys.ini]: We need to escape the `"` values

**Comment by zero-24** [comment]
Great thanks, please take a look into my suggestion. Once that is merged I will do a release in the comming days with the new language file.

**Comment by zero-24** [comment]
Thanks will merge it now and prepare the release in the comming days but could take until the weekend.

**Comment by AlainRnet** [comment]
off topic: I started to use github for my own development and I saw your script in the build folder to build the zip file and I was wondering how you were doing the release.
If I understand correctly, your script builds the zip to the root but I see that the link in the update.xml file points to releases/download/3-5/plg_system_customcss.zip

We can contact each other in another way or continue the discussion here, if you agree. Thanks again!

**Comment by AlainRnet** [comment]
I noticed that you were busy before the weekend, so am I. There is no emergency and I will read you to your best convenience.

**Comment by zero-24** [comment]
> off topic: I started to use github for my own development and I saw your script in the build folder to build the zip file and I was wondering how you were doing the release.

- I do checkout the latest version
- locally change with the cli to the root folder of this repo
- run the build script
- commit the changes
- tag the release
- upload the tag to github
- create & publish the release on github
- push the commits

In the case we do have now I will also run a sync to crowdin so that your translation is uploaded there too.

> If I understand correctly, your script builds the zip to the root but I see that the link in the update.xml file points to releases/download/3-5/plg_system_customcss.zip

The package will be placed within the build folder. (https://github.com/zero-24/plg_system_customcss/blob/master/build/build.sh#L2) The link in the update.xml is comming from the github release. As I'm not going to commit the build zip package. That file is only uploaded to the release.

**Comment by zero-24** [comment]
The release is out thanks @AlainRnet : https://github.com/zero-24/plg_system_customcss/releases/tag/3-7

**Comment by AlainRnet** [comment]
Oh, I didn't see you was on Crowdin. I'll see to translate them on https://joomla.crowdin.com/zero-24
And thank you very much for describing your release process!

**Pull request merged**